src/sentry/static/sentry/app/views/settings/organization/general/organizationGeneralSettingsView.jsx

```diff
@@ -1,8 +1,8 @@
 import {browserHistory} from 'react-router';
 import PropTypes from 'prop-types';
 import React from 'react';
-import createReactClass from 'create-react-class';
 import Reflux from 'reflux';
+import createReactClass from 'create-react-class';
```

Comment:
looks like this page doesn't have jest tests? are those hard to add for the new components (e.g. just a snapshot that has mocked api for rendering)?

Reply:
Yeah I'll add them in this PR https://github.com/getsentry/sentry/pull/7303/files